common/lib/client/auth.ts

```diff
@@ -10,10 +10,11 @@ import HmacSHA256 from 'crypto-js/build/hmac-sha256';
 import { stringify as stringifyBase64 } from 'crypto-js/build/enc-base64';
 import { createHmac } from 'crypto';
 import { ErrnoException, RequestCallback, RequestParams } from '../../types/http';
-import TokenDetails from '../../types/TokenDetails';
-import TokenParams from '../../types/TokenParams';
+import {Types} from '../../../ably';
```

Comment:
We should try and keep the styling of these consistent with the existing imports

Reply:
Changed to `import * as API` as we talked about